### DISCUSSION TITLE
Installing Casadi in Anaconda with Gurobi (Windows)

### DISCUSSION THREAD

**Discussion by B-Kuma** [casadi/casadi#3512] [General]
Hello everybody,

I'm trying to use casadi with anaconda and need access  to the gurobi solver on Windows.
I already installed casadi 3.6.3 with:

> conda install casadi -c conda-forge

And I installed gurobi 11.0.0 with:

> conda install gurobi -c gurobi

I also installed the gurobi software (Version 11.0.0) from gurobis web-site and connected it with an academic license from my university

But when I'm trying to use gurobi as a solver it can't find casadi_conic_gurobi.dll, and there is also nowhere such a file. Maybe i looked in the wrong directories. 

> \anaconda3\Lib\site-packages\casadi

and 

> \anaconda3\pkgs\casadi-3.6.3-py311h8ed2231_3

In those directories isn't even one file or subdirecory containing gurobi in it's name.

I really don't know how to move on from here. 

I'm thankful for any help.

Kind regards

**Comment by jgillis** [comment]
The gurobi interface is shipped in the official casadi pypi wheels.

The conda package ( https://github.com/conda-forge/casadi-feedstock ) is maintained by the community.

I gave some [suggestion](https://github.com/conda-forge/casadi-feedstock/pull/84#issuecomment-1498976704) to enable commercial solvers plugins for the feedstock (should be very simple in 3.6), but I don't think it was implemented.

**Reply by traversaro** [reply]
Sorry, I did not saw this discussion, @B-Kuma please open an issue at https://github.com/conda-forge/casadi-feedstock if you have a feature request on the conda-forge packages of casadi, thanks!
